internal/pkg/backend/local/local.go

```diff
@@ -1,13 +1,15 @@
 package local
 
 import (
+	"errors"
 	"io/ioutil"
+	"log"
 	"os"
 	"path/filepath"
 	"sync"
 )
 
-const DefaultStateFile = ".devstream/devstream.state"
+const DefaultStateFile = "devstream.state"
```

Comment:
Our previous plan was to put this file in the `.devstream/` directory, why did we decide to change it?

Reply:
See the description for the details.

TL;DR: when specifying a plugin directory with "-p", this folder won't be created so there is an error when creating the state file.